lib/commands/ping.js

```diff
@@ -0,0 +1,43 @@
+'use strict'
+
+const npmConfig = require('../config.js')
+const libnpm = require('libnpm')
+const figgyPudding = require('figgy-pudding')
+const log = require('npmlog')
+
+module.exports = ping
+
+function ping (argv) {
+  const {json, registry, silent} = argv
+
+  const PingConfig = figgyPudding({
+    json: {
+      default: json
+    },
+    registry: {
+      default: registry
+    }
+  })
+
+  const opts = PingConfig(npmConfig())
+  log.notice('PING', registry)
+  const start = Date.now()
+  return libnpm.fetch('/-/ping?write=true', opts).then(res => {
```

Comment:
`npm-registry-fetch` has a nice little `.json()` method that can make this more concise:
```suggestion
  return libnpm.fetch.json('/-/ping?write=true', opts).catch(() => ({})).then(res => {
```